sm/download/src/download.rs

```diff
@@ -282,7 +288,10 @@ mod tests {
 
         let downloader = Downloader::new(&name, &version, target_dir_path.path());
         match downloader.download(&url).await {
-            Ok(()) => return Ok(()),
+            Ok(()) => {
+                assert_eq!("".as_bytes(), std::fs::read(downloader.filename())?);
```

Comment:
The test would be more convincing with a non empty file.

Reply:
There is already a test for that.